### ISSUE TITLE
Creating kubernetes vpc cluster timeout even if cluster is up and running

### ISSUE THREAD

**Issue by Mallear** [IBM-Cloud/terraform-provider-ibm#1210]
Hi there,

### Terraform Version
Terraform v0.12.23
+ provider.ibm v1.2.5

### Affected Resource(s)
- ibm_container_vpc_cluster

### Terraform Configuration Files
```hcl
resource "ibm_container_vpc_cluster" "k8s_cluster" {
  resource_group_id = data.ibm_resource_group.resource_group.id
  name              = "k8s-cluster"
  vpc_id            = var.vpc
  flavor            = var.instance_flavor

  worker_count = var.worker_count
  kube_version = var.kube_version

  dynamic "zones" {
    for_each = var.subnets
    content {
      name      = zones.key
      subnet_id = zones.value
    }
  }

  tags = local.tags
}

```

### Debug Output
```
module.cluster.ibm_container_vpc_cluster.k8s_cluster: Still creating... [1h29m30s elapsed]
module.cluster.ibm_container_vpc_cluster.k8s_cluster: Still creating... [1h29m40s elapsed]
module.cluster.ibm_container_vpc_cluster.k8s_cluster: Still creating... [1h29m50s elapsed]
module.cluster.ibm_container_vpc_cluster.k8s_cluster: Still creating... [1h30m0s elapsed]
Error: timeout while waiting for state to become 'Ready' (last state: 'Deploy in progress', timeout: 1h30m0s)
  on ../../../modules/k8s/main.tf line 9, in resource "ibm_container_vpc_cluster" "k8s_cluster":
   9: resource "ibm_container_vpc_cluster" "k8s_cluster" {
```

### Expected Behavior
Terraform create next resources when cluster is ready.

### Actual Behavior
Terraform timeout after 1h30 waiting for cluster to be ready even if the cluster is up and running in IBM console.

### Steps to Reproduce
1. `terraform apply`

**Comment by hkantare** [comment]
Here cluster we wait until  the ingress is also assigned..We provided a attribute to control the creation of cluster using wait_till parameter please refer to docs for more info.Can you confirm the ingress is assigned?
https://cloud.ibm.com/docs/terraform?topic=terraform-container-resources#vpc-cluster-input

**Comment by Mallear** [comment]
Ok it does work with `wait_till = "OneWorkerNodeReady"`, but not with `IngressReady` as my load balancer take waaaaaay longer to be ready .... Like it's been 2h and it's still in creating state ...

Anyway, `wait_till` parameter is available since 1.2.4 and I didn't have timeouts before upgrading to 1.2.5 :/

**Comment by Mallear** [comment]
Problems seems to come from IBM side. I've managed to deploy a cluster this afternoon after a failure this morning due to worker network issue.
